components/mediation-ui/org.wso2.carbon.rest.api.ui/pom.xml

```diff
@@ -89,6 +89,20 @@
             <groupId>org.wso2.carbon</groupId>
             <artifactId>org.wso2.carbon.logging</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.mediation</groupId>
+            <artifactId>org.wso2.carbon.mediation.transport.handlers</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.commons</groupId>
+            <artifactId>org.wso2.carbon.tenant.common</artifactId>
+            <version>4.6.43</version>
```

Comment:
Shall we get the version from parent pom

Reply:
Done